stuff/cron/testing/test_plagiarism_detector.py

```diff
@@ -34,7 +27,10 @@
     "00000000000000000000000000000007", "00000000000000000000000000000008",
     "00000000000000000000000000000009"
 ]
-LOCAL_DOWNLOADER_DIR = "/opt/omegaup/stuff/cron/testing/testdata/"
+
+_OMEGAUP_ROOT = os.path.abspath(os.path.join(__file__, '..'))
+LOCAL_DOWNLOADER_DIR = os.path.join(_OMEGAUP_ROOT, "testdata", "")
```

Comment:
```suggestion
LOCAL_DOWNLOADER_DIR = os.path.join(_OMEGAUP_ROOT, "testdata")
```

the last component is unneeded

Reply:
Done.